### DISCUSSION TITLE
Add stablecoins to default accounts

### DISCUSSION THREAD

**Discussion by pclaesen** [ConsenSys-archive/truffle#5515] [Ideas]
When launching Ganache, you automatically get 10 accounts with 1000 ETH.
I'm still looking for an easy way to get DAI, USDC in those wallets.
Would be nice to automatically have DAI and USDC, USDT,... in those wallet when launching.

I have no idea about the technical things that need to happen before this is possible, just figured it'd be a nice addition.

**Comment by kevinbluer** [comment]
Nice idea @pclaesen! cc @davidmurdoch @MicaiahReid for their respective radars 👀

**Reply by davidmurdoch** [reply]
@pclaesen Can you open a new issue on the the github.com/trufflesuite/ganache repo to request this as a feature?

**Reply by pclaesen** [reply]
https://github.com/trufflesuite/ganache/issues/3653
Fingers crossed:-D.
